src/azul/project/hca/config.py

```diff
@@ -29,6 +29,21 @@ def __init__(self, dss_url: str, es_endpoint: Tuple[str, int]) -> None:
                             }
                         }
                     }
+                },
```

Comment:
This seems odd. Why do we have to make a special case in the mapping for `total_estimated_cells`?

Reply:
Take into account that I know very little about ES. It just seems weird to me that one field get special handling. Is it the only numeric facet?

Reply:
> We might want to make it less "special" by matching all long types instead of just total_estimated_cells. I don't expect any side-effects doing so, yet it will be ready if we decide to add new numeric facet.

I like that! Can you do it in this PR?

Reply:
Sure :)